src/App.vue

```diff
@@ -314,7 +314,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   get feeSummaryConfirmLabel (): string {
     const isNoFee = this.isFirmChangeFiling || this.isFirmConversionFiling
     if (this.isSummaryMode) {
-      return (isNoFee && !this.getFilingData.priority) ? 'File Now (No Fee)' : 'File and Pay'
+      return (isNoFee && !this.getFilingData.some(fd => fd.priority)) ? 'File Now (No Fee)' : 'File and Pay'
```

Comment:
[Lines 48-65](https://github.com/bcgov/business-edit-ui/blob/8bbfb5c46b40c4b4013b99b3ddf05430fce0a449/src/mixins/fee-mixin.ts#L48-L65) maps the priority and waveFees from getStaffPayment. If priority is false from getStaffPayment, then each priority in the array is false. While this way works too, we only need to verify one priority.